spring-cloud-contract-verifier/src/main/groovy/org/springframework/cloud/contract/verifier/config/framework/TestFrameworkDefinition.groovy

```diff
@@ -0,0 +1,31 @@
+package org.springframework.cloud.contract.verifier.config.framework
+
+/**
+ * Defines elements characteristic of a given test framework to be used during test class construction.
+ *
+ * @author Olga Maciaszek-Sharma
+ */
+interface TestFrameworkDefinition {
```

Comment:
`@since`, `@CompileStatic` + javadocs on methods with an example. 

E.g. (I don't know if that's correct with `{@code ;}`)

```groovy
/**
* Returns the characters that should end each line. E.g. for Java tests that would mean {@code ;}
**/
String getLineSuffix() 
```